### PR TITLE
Switch resources to posts

### PR DESCRIPTION
**Pull request by SondraE**
## Steps to test/reproduce
- "Posts" in nav
- "Posts by date" /blog/
- "Posts by tag" /tags/
- "2022 Posts" banner /blog/
- "Posts" banner on author and tools detail pages

### CODE REVIEW

**Comment by SondraE** [comment]
@stacyk (Not urgent) I switched the term from "resources" to "posts" where I was confident I was only changing displayed text. Are there other places you recommend changing this term?

**Review by jgerigmeyer — APPROVED** [review]
I added a few more; lgtm! 👍

**Comment by jgerigmeyer** [comment]
[Reconsider 'Resources' nav term](https://trello.com/c/Z9qzS5F5/178-reconsider-resources-nav-term)

**Comment by SondraE** [comment]
[Reconsider 'Resources' nav term](https://trello.com/c/Z9qzS5F5/178-reconsider-resources-nav-term)

**Comment by stacyk** [comment]
@jgerigmeyer I am guessing these need to re-index, but I'm not sure how that works. 
Screen Shot 2022-11-14 at 3 47 35 PM

**Comment by jgerigmeyer** [comment]
> @jgerigmeyer I am guessing these need to re-index, but I'm not sure how that works.

I think it happens via Netlify on every production build.

File: content/_includes/post.macros.njk [hunk 1]
```diff
@@ -262,41 +262,6 @@ params:
 {% endmacro %}
 
 
-{# @docs
```
Comment (stacyk) [inline content/_includes/post.macros.njk:265]: @jgerigmeyer I take it we are not using this anywhere (anymore at least)?

**Review by stacyk — APPROVED** [review]
This looks good. I don't think the re-indexing (or whatever the search needs) should hold this up.

File: content/_includes/post.macros.njk (hunk 1 above)
Comment (jgerigmeyer) [reply content/_includes/post.macros.njk:265]: Yeah, I couldn't find any uses of it, and the build didn't error when I removed it. 🤷

**Pull request merged**